portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/APIDefinition/APIDefinition.jsx

```diff
@@ -726,7 +726,7 @@ class APIDefinition extends React.Component {
                                 />
                             </Button>
                         ) : (
-                            !(graphQL || api.type === API.CONSTS.APIProduct) && (
+                            !(graphQL || api.apiType === API.CONSTS.APIProduct) && (
```

Comment:
isAPIProduct is passed down the context. You can use it as follows.
Update line 164 as follows and use it to do the comparison.
```
const { settings, isAPIProduct } = this.context;
```

Reply:
It seems like there is no value contained in the 'context' for isAPIProduct

Reply:
Since the api.apiType is coming as null all the time we can't rely on that..
```
     const { 
            match,
        } = this.props;
const isApiProduct = match.path.search('/api-products/') !== -1 ;
```
We can use above logic to detect if an API is API product or not.

Reply:
I have done this change and it works fine. Thank you